static/js/controllers/reports-add.js

```diff
@@ -17,6 +18,10 @@ angular.module('inboxControllers').controller('ReportsAddCtrl',
 
     var getSelected = function() {
       if ($state.params.formId) { // adding
+        $scope.geolocationWatcher = Geolocation.watch(function(position) {
```

Comment:
I don't think this needs to be on the $scope does it? Just a plain old `var` will do.